src/sst/core/Makefile.am

```diff
@@ -15,7 +15,7 @@ AM_CPPFLAGS =  \
 
 DIST_SUBDIRS = libltdl .
 SUBDIRS = libltdl .
-EXTRA_DIST = mainpage.dox eli/README.md
+EXTRA_DIST = sstinfo.h mainpage.dox eli/README.md
```

Comment:
It may seem couterintuitive, but if you want a header file to be part of make dist, but not to be installed, you should add it to the "sst_core_sources" section of Makefile.am.